### PR TITLE
publish: Memfault esp component article

### PR DESCRIPTION
**Pull request by gminn**
### Summary

 We have a new article about creating Memfault's new ESP component 🥳

 ### Test Plan

  - [x] Check the article in the preview
  - [x] Validate links and images

### CODE REVIEW

**Comment by gminn** [comment]
* **#529** <a href="https://app.graphite.dev/github/pr/memfault/interrupt/529?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gminn and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: _posts/2024-10-24-memfault-esp-component.md [hunk 1]
```diff
@@ -215,10 +216,11 @@ change, we'll create a bind mount with the correct name
 component build without needing to copy files around:
 
 ```bash
-mkdir /tmp/memfault-firmware-sdk && sudo mount --bind /path/to/memfault-monorepo/sdk/embedded /tmp/memfault-firmware-sdk
+mkdir /tmp/memfault-firmware-sdk && \
```
Comment (gminn) [inline _posts/2024-10-24-memfault-esp-component.md:219]: So readers don't have to scroll!

File: _data/authors.yml [hunk 2]
```diff
@@ -88,7 +88,7 @@ noah:
   github: noahp
   blurb:
     is an embedded software engineer at Memfault. Noah previously worked on
-    embedded software teams at Fitbit and Markforged
+    embedded software teams at Fitbit and Markforged.
```
Comment (noahp) [inline _data/authors.yml:91]: :smile: tyty

File: _posts/2024-10-24-memfault-esp-component.md [hunk 3]
```diff
@@ -3,6 +3,7 @@ title: Publishing the Memfault SDK as an ESP-IDF Component
 description: How we shipped our SDK as an ESP-IDF component
 author: noah
 tags: [ci, memfault, esp32, esp-idf, github-actions]
+image: /img/memfault-esp-component/esp-registry-page.png
```
Comment (noahp) [inline _posts/2024-10-24-memfault-esp-component.md:6]: <3

File: _posts/2024-10-24-memfault-esp-component.md [hunk 4]
```diff
@@ -32,7 +33,7 @@ A little over 2 years ago, in 2022, Espressif introduced the
 [ESP Component Registry](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-component-manager.html),
 to streamline sharing and reusing public components.
 
-Initially it was used mostly for breaking up the ESP-IDF itself into smaller
+Initially, it was primarily used for breaking up the ESP-IDF itself into smaller
```
Comment (noahp) [inline _posts/2024-10-24-memfault-esp-component.md:36]:
```suggestion
Initially, it was used for breaking up the ESP-IDF itself into smaller
```

File: img/memfault-esp-component/esp-registry-page.png
Comment (noahp) [inline img/memfault-esp-component/esp-registry-page.png:1]: amazing, thank you for adding!

**Review by noahp — APPROVED** [review]
Thank you Gilly!

**Pull request merged**